app/src/main/res/layout/fragment_video_preview.xml

```diff
@@ -25,18 +25,63 @@
                 android:layout_width="wrap_content"
                 android:layout_height="wrap_content"
                 android:layout_alignParentStart="true"
-                android:layout_alignParentLeft="true"
                 android:layout_alignParentTop="true"
                 android:textColor="@color/text_main"
                 android:textSize="24sp" />
 
+            <TextView
+                android:id="@+id/textSubtitles"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:layout_below="@+id/textTitle"
+                android:ellipsize="marquee"
+                android:singleLine="true"
+                android:textColor="@color/text_second"
+                android:textSize="14sp"
+                android:visibility="gone" />
+
+            <TextView
+                android:id="@+id/headerTextPreviewDescription"
+                android:layout_width="fill_parent"
+                android:layout_height="48dp"
+                android:layout_below="@+id/textSubtitles"
+                android:layout_alignParentStart="true"
+                android:background="@color/transparent"
+                android:fontFamily="sans-serif-medium"
+                android:gravity="center_vertical"
+                android:text="@string/title_section_description"
+                android:textColor="@color/apptheme_secondary"
+                android:textSize="14sp" />
+
+            <TextView
+                android:id="@+id/textPreviewDescription"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_below="@+id/headerTextPreviewDescription"
+                android:layout_alignParentStart="true"
+                android:ellipsize="marquee"
+                android:foreground="@drawable/gradient_light_to_transparent_from_bottom"
+                android:maxLines="3"
+                android:textSize="14sp" />
+
+            <TextView
+                android:id="@+id/showDescriptionButton"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_below="@+id/textPreviewDescription"
+                android:layout_alignParentEnd="true"
+                android:paddingTop="8dp"
```

Comment:
I personally would use a smaller margin (not padding, because it makes the view bigger) to the top. I feel the distance between the button and view is quite big